File: src/main/java/pl/mroz/buddiesapi/interfaces/rest/OpenApiConfig.java
package pl.mroz.buddiesapi.interfaces.rest;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class OpenApiConfig {
    @Bean
    public OpenAPI buddiesAPI() {
        return new OpenAPI()
                .info(new Info().title("Buddies API")
                        .description("")
                        .version("v0.0.1")
                );
    }
}

File: src/main/kotlin/buddiesapi/interfaces/rest/OpenApiConfig.kt
package buddiesapi.interfaces.rest

import io.swagger.v3.oas.models.OpenAPI
import io.swagger.v3.oas.models.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class OpenApiConfig {
    @Bean
    fun buddiesAPI(): OpenAPI {
        return OpenAPI()
            .info(
                Info().title("Buddies API")
                    .description("")
                    .version("v0.0.1")
            )
    }
}
